plugins/inventory/ovirt.py

```diff
@@ -39,37 +41,42 @@
         env:
           - name: OVIRT_PASSWORD
       ovirt_cafile:
-        description: path to ovirt-engine CA file.
+        description: path to ovirt-engine CA file. If C(ovirt_cafile) parameter is not set and C(ovirt_insecure) is not True, system wide CA certificate store\
+        is used.
         required: False
+      ovirt_insecure:
+        description: A boolean flag that indicates if the server TLS certificate and host name should be checked.
+        required: False
+        default: False
       ovirt_query_filter:
-        required: false
+        required: False
         description: dictionary of filter key-values to query VM's. See U(https://ovirt.github.io/ovirt-engine-sdk/master/services.m.html#ovirtsdk4\
 .services.VmsService.list) for filter parameters.
       ovirt_hostname_preference:
-        required: false
+        required: False
         description: list of options that describe the ordering for which hostnames should be assigned. See U(https://ovirt.github.io/ovirt-engin\
 e-api-model/master/#types/vm) for available attributes.
         default: ['fqdn', 'name']
         type: list
 '''
 
 EXAMPLES = '''
-# Sample content of ovirt.yml
-# plugin: ovirt
-# ovirt_url: http://localhost/ovirt-engine/api
-# ovirt_username: ansible-tester
-# ovirt_password: secure
-# ovirt_query_filter:
-#   search: 'name=myvm AND cluster=mycluster'
-#   case_sensitive: no
-#   max: 15
-# keyed_groups:
-#   - key: cluster
-#     prefix: 'cluster'
-# groups:
-#   dev: "'dev' in tags"
-# compose:
-#   ansible_host: devices["eth0"][0]
+ # Sample content of ovirt.yml
+ plugin: ovirt
+ ovirt_url: http://localhost/ovirt-engine/api
```

Comment:
Can we add here `https://` and `ca_file` parameter(and maybe comment how to get the CA file(wget https://engine/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA)) so we provide the best practise in the examples?